includes/Order/Hooks.php

```diff
@@ -328,10 +328,6 @@ public function ensure_vendor_coupon( $valid, $coupon, $discount ) {
 
         $available_vendors = array_unique( $available_vendors );
 
-        if ( $coupon->is_type( 'fixed_cart' ) && count( $available_vendors ) > 1 ) {
```

Comment:
removing the code doesn't help. The calculation will be wrong. You can't apply fixed cart discount on multiple vendor products.